### DISCUSSION TITLE
Your database schema does not match your current configuration

### DISCUSSION THREAD

**Discussion by deepanaresh** [vendurehq/vendure#1996] [Q&A] [answered]
Hi, I have just setup vendure on my dev machine with Postgres DB and when I add the following code in vendure-config.ts and compile 

customFields: {
        Product: [
            { name: 'weight', type: 'int', defaultValue: 0 }
        ],
        Customer: [

            { name: 'avatar', type: 'relation', entity: Asset }
        ]
    },

I get the following error :

Your database schema does not match your current configuration. 

Any help would be highly appreciated  @michaelbromley

Thank you
Deepa

**Comment by deepanaresh** [comment, marked as answer]
Sorry Guys, My bad since I was new to the Vendure / NodeJS missed the obvious.   I have resolved the issue now

**Reply by michaelbromley** [reply]
Glad you got it sorted. In case anyone else runs into this - what is missing is to run a migration: https://www.vendure.io/docs/developer-guide/migrations/

**Reply by deepanaresh** [reply]
Thanks Micheal
